src/redux/books/booksSlice.js

```diff
@@ -1,26 +1,45 @@
 /* eslint-disable no-param-reassign */
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [
-    {
-      id: 1,
-      title: 'The name of faith',
-      author: 'Sooner',
-    },
-    {
-      id: 2,
-      title: 'Contradiction paths',
-      author: 'HABASM',
-    },
-    {
-      id: 3,
-      title: 'Some of the path life',
-      author: 'Durbete Gojam',
-    },
-  ],
+  books: [],
+  status: 'idle',
+  error: '',
 };
 
+// Async thunk to fetch books
+export const fetchBooks = createAsyncThunk('books/fetchBooks', async () => {
+  const response = await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${process.env.REACT_APP_APP_ID}/books`);
```

Comment:
- [ ] Nice work making API calls however in the requirements it is said that all HTTP requests should be done using the `AXIOS` library. Kindly work on that.